README.md

```diff
@@ -19,6 +19,36 @@ Run the services locally with Docker Compose
 $ docker-compose up
 ```
 
+### Set up AWS SQS queue locally
+
+Run AWS services locally using localstack.
```

Comment:
Would you be able to explain the technical architecture before hand, e.g. the data gets sent to a AWS Queue in DECs service.

Reply:
Okay yeah I can do that. Max and I were gonna make some changes to the README anyway so we can add more about that.

Reply:
thanks 👍

Reply:
@MaxOSullivan

Reply:
I don't think you'll have time for this, so maybe just add this as part of a tech debt ticket for the decs intergration